src/types/phone.js

```diff
@@ -0,0 +1,22 @@
+const phone = {
+  default: context => {
+    const defaultPhoneRegEx = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/
```

Comment:
Thinking about it more, since we're making `numeric` available, then maybe this makes sense. But I'd switch the regex to use a backreference on the divider so that you don't allow people to use a dot in one place and a dash in another. Like:

`/^\(?\d{3}\)?([-. ]?)\d{3}\1\d{4}$/`

(the above is untested)